src/manifolds/Sphere.jl

```diff
@@ -114,6 +114,16 @@ metric from the embedding, i.e. $ (v,w)_x = v^\mathrm{T}w $.
 """
 @inline inner(S::Sphere, x, w, v) = dot(w, v)
 
+function inverse_represent_in_basis(M::Sphere{N}, x, v, B::ArbitraryOrthonormalBasis) where N
+    if isapprox(x[end], -1)
```

Comment:
This is a minor point, but I prefer that the privileged point be `(1,...,0)`, not `(0,...,-1)`. On most spheres, there's no reason to prefer one point over another, but for `Sphere(1)` and `Sphere(3)` identified as the circle and compact symplectic groups, the point `(1,...,0)` is privileged. For consistency, I think that point should be privileged for all spheres.